app/views/static_pages/kontakt.html.haml

```diff
@@ -27,18 +20,18 @@
   %a{href: "https://github.com/heartsofcode"} heartsofcode
 
 %p.spacer-m
-  IRL: Skalitzer Str. 100, 10997 Berlin-Kreuzberg
+  IRL: Mariannenplatz 2 A, 10997 Berlin-Kreuzberg
```

Comment:
It says "Marielle-Franko-Platz" on the sign now, we should mention both I think, the 'official' and the current name